test/test_jit_expression.py

```diff
@@ -141,3 +142,63 @@ def test_rank1(compile_args):
     u_correct = np.array([f[1], f[0]]) + gradf0
 
     assert np.allclose(u_ffcx, u_correct.T)
+
+
+def test_elimiate_zero_tables_tensor(compile_args):
```

Comment:
Not clear how this is being 'tested'. Does it rely on struct compiler flags picking up the issue?

Reply:
It is not needed. I'll remove it

Reply:
What is not needed? All tests which call JIT take compile_args fixture. Thats how we control JIT compilation parameters, especially it uses "-O1" flag to speed them up.